query/src/configs/config_meta.rs

```diff
@@ -71,6 +76,7 @@ pub struct MetaConfig {
 impl MetaConfig {
     pub fn default() -> Self {
         MetaConfig {
```

Comment:
Cool!
meta_embedded_dir is a test purpose path, it would be better to set it as a random uuid instead config it in the config file?
If so, We can improve it in the another PR
cc @drmingdrmer